modules/connatixBidAdapter.js

```diff
@@ -128,12 +155,13 @@ export const spec = {
       cpm: bidResponse.Cpm,
       ttl: bidResponse.Ttl || DEFAULT_MAX_TTL,
       currency: 'USD',
-      mediaType: BANNER,
+      mediaType: bidResponse.VastXml ? VIDEO : BANNER,
```

Comment:
What about adpod?

Reply:
We do not suport adpod mediatype. We filtered it out in the is `isBidRequestValid` method.